File: CS/Form1.cs
using DevExpress.XtraPrinting;
using DevExpress.XtraReports.UI;
using System.Windows.Forms;
using System;

namespace XlsxExportExample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            // Create a report.
            XtraReport report = new XtraReport() {
                Name = "Report Example",
                Bands = {
                    new DetailBand() {
                        Controls = {
                            new XRLabel() {
                                Text = "Some content goes here...",
                            }
                        }
                    }
                }
            };

            // Specify export options.
            XlsxExportOptions xlsxExportOptions = new XlsxExportOptions() {
                ExportMode = XlsxExportMode.SingleFile,
                ShowGridLines = true,
                FitToPrintedPageHeight = true
            };

            // Specify the path for the exported XLSX file.  
            string xlsxExportFileName =
                Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) +
                @"\Downloads\" +
                report.Name +
                ".xlsx";

            // Export the report.
            report.ExportToXlsx(xlsxExportFileName, xlsxExportOptions);
            System.Diagnostics.Process.Start(xlsxExportFileName);
        }
    }
}

File: VB/Form1.vb
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraReports.UI
Imports System.Windows.Forms
Imports System

Namespace XlsxExportExample

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            ' Create a report.
            Dim report As XtraReport = New XtraReport() With {.Name = "Report Example"}
            ' Specify export options.
            Dim xlsxExportOptions As XlsxExportOptions = New XlsxExportOptions() With {.ExportMode = XlsxExportMode.SingleFile, .ShowGridLines = True, .FitToPrintedPageHeight = True}
            ' Specify the path for the exported XLSX file.  
            Dim xlsxExportFileName As String = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) & "\Downloads\" & report.Name & ".xlsx"
            ' Export the report.
            report.ExportToXlsx(xlsxExportFileName, xlsxExportOptions)
            System.Diagnostics.Process.Start(xlsxExportFileName)
        End Sub
    End Class
End Namespace
